### ISSUE TITLE
No relation between vmachine and vdisk after creating virtual machine on KVM (virt-install)

### ISSUE THREAD

**Issue by matthiasdeblock** [openvstorage/framework#594]
After creating a virtual machine with KVM (virt-install) there is no relation between vmachine and vdisk. 

```
openvstorage:
  Installed: 2.7.0-rev.3556.7283113-1
```

```
2016-06-09 15:17:49 48300 +0200 - cmp04 - 24265/140712111441728 - celery/celery.pool - 15328 - DEBUG - TaskPool: Apply <function _fast_trace_task at 0x7ffa12f82cf8> (args:('ovs.machine.update_from_voldrv', 'ca920dbb-c200-41b2-8d21-06e77fe4bc01', [], {'storagedriver_id': u'myvpoolDCA2L4E3TFL83Br4', 'name': u'/DCA2L4E3TFL83Br4/VM_2515.xml'}, {'utc': True, u'is_eager': False, 'chord': None, u'group': None, 'args': [], 'retries': 0, u'delivery_info': {u'priority': None, u'redelivered': False, u'routing_key': u'sr.DCA2L4E3TFL83Br4', u'exchange': u'generic'}, 'expires': None, u'hostname': 'celery@cmp04', 'task': 'ovs.machine.update_from_voldrv', 'callbacks': None, u'correlation_id': u'ca920dbb-c200-41b2-8d21-06e77fe4bc01', 'errbacks': None, 'timelimit': (None, None), 'taskset': None, 'kwargs': {'storagedriver_id': u'myvpoolDCA2L4E3TFL83Br4', 'name': u'/DCA2L4E3TFL83Br4/VM_2515.xml'}, 'eta': '2016-06-09T13:17:49.472955+00:00', u'reply_to': u'e1edd6d9-73c0-314f-af60-39a76547cacc', 'id': 'ca920dbb-c200-41b2-8d21-06e77fe4bc01', u'headers': {}}) kwargs:{})
2016-06-09 15:17:49 76500 +0200 - cmp04 - 20485/140712111441728 - celery/celery.redirected - 15315 - WARNING - 2016-06-09 15:17:49 76400 +0200 - cmp04 - 20485/140712111441728 - log/volumedriver_task - 15314 - INFO - [ovs.lib.vmachine.update_from_voldrv] - [] - {"storagedriver_id": "myvpoolDCA2L4E3TFL83Br4", "name": "/DCA2L4E3TFL83Br4/VM_2515.xml"} - {"storagedriver": "b5ebe200-4734-407a-9742-f914d0580032"}
2016-06-09 15:17:52 17300 +0200 - cmp04 - 20485/140712111441728 - celery/celery.redirected - 15321 - WARNING - 2016-06-09 15:17:52 17300 +0200 - cmp04 - 20485/140712111441728 - extensions/volatile mutex - 15320 - WARNING - A lock on ovs_lock_DCA2L4E3TFL83Br4/VM_2515.xml_none was kept for 0.300060033798 sec
2016-06-09 15:17:52 44000 +0200 - cmp04 - 20485/140712111441728 - lib/vmachine - 15324 - INFO - Syncing vMachine (device DCA2L4E3TFL83Br4/VM_2515.xml, ip 192.168.122.1, mountpoint /mnt/myvpool)
2016-06-09 15:17:54 46400 +0200 - cmp04 - 20485/140712111441728 - lib/vmachine - 15335 - INFO **- Updating vMachine finished (name VM_2515, 0 vdisks (re)linked)**
2016-06-09 15:17:54 46500 +0200 - cmp04 - 20485/140712111441728 - celery/celery.redirected - 15337 - WARNING - 2016-06-09 15:17:54 46500 +0200 - cmp04 - 20485/140712111441728 - extensions/volatile mutex - 15336 - WARNING - A lock on ovs_lock_DCA2L4E3TFL83Br4/VM_2515.xml_none was kept for 2.29092502594 sec
```

**Workaround**
Change the code in /opt/OpenvStorage/ovs/lib/vmachine.py from: 
`datastores = dict([('{0}:{1}'.format(storagedriver.storage_ip, storagedriver.mountpoint), storagedriver) for storagedriver in storagedrivers])`
To: 
`datastores = dict([('127.0.0.1:{0}'.format(storagedriver.mountpoint), storagedriver) for storagedriver in storagedrivers])`

**Comment by wimpers** [comment]
@matthiasdeblock  please create a PR for it

**Comment by wimpers** [comment]
I believe this might be related to code @khenderick is tackling atm. It should use the storage IP not 127.0.0.1

**Comment by eduardBM** [comment]
Testing Fix (1.5 h)

**Comment by eduardBM** [comment]
openvstorage-2.7.0_rev.3582.80636a4-1
